src/test/java/edu/hm/hafner/util/SetTest.java

```diff
@@ -0,0 +1,169 @@
+package edu.hm.hafner.util;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.*;
+import java.util.HashSet;
+
+/**
+ * Tests the class HashSet.
+ *
+ * @author Sofia Steinleitner
+ */
+class SetTest {
+    private static final Integer eins = 1;
+    private static final Integer zwei = 2;
+    private static final Integer drei = 3;
+    private static final Integer vier = 4;
+
+    @Test
+    void shouldTestConstructorHashSet() {
+        HashSet<Integer> set = new HashSet<Integer>();
+        Assertions.assertThat(set).isEmpty();
```

Comment:
```suggestion
        assertThat(set).isEmpty();
```
In einer Testklasse am besten einen statischen Import für `Assertions` verwenden.